crdppf/views/printproxy.py

```diff
@@ -113,9 +113,21 @@ def status(self):
     def report_get(self):
         """ Get the PDF. """
 
-        return self._proxy_response(
+        try:
+            archive_path = self.config['pdf_archive_path']
+        except: 
+            archive_path = None
+
+        pdf = self._proxy_response(
             "%s/report/%s" % (
                 self.config['print_url'],
                 self.request.matchdict.get('ref')
             ),
         )
+        
+        if archive_path is not None:
+            filename = pdf.content_disposition.split('=')[1]

```

Comment:
or is there a better way to get the filename I didn't see?

Reply:
Uh... I do not like it, but I cannot figure out another way.

What does the PDF name look like ? Would it be possible to use something which comes from `self.request.params` ?

Reply:
Probably we could, but we want to keep the initial creation date. I might improve that later on